domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -200,6 +207,14 @@ public LocalDate getServiceDate() {
         return TotalAmountCalculator.calculateInterestForClaim(this);
     }
 
+    public Optional<ClaimantResponse> getClaimantResponse() {
+        return Optional.ofNullable(claimantResponse);
+    }
+
+    public LocalDateTime getClaimantRespondedAt() {
```

Comment:
shouldn't this be optional as well?

Reply:
Pattern seems to be these dates aren't optional

Reply:
no, i mean look at all the others

Reply:
we might need to clean others as well at some point. prefer to use `Optional` instead of returning `null`